core-domain/src/test/java/org/uniprot/core/uniparc/impl/UniParcCrossReferenceImplTest.java

```diff
@@ -27,7 +28,14 @@ void builderFrom_constructorImp_shouldCreate_equalObject() {
                         -3,
                         true,
                         LocalDate.now(),
-                        LocalDate.now());
+                        LocalDate.now(),
+                        "geneName",
+                        "proteinName",
+                        new OrganismBuilder().taxonId(10L).build(),
+                        "chain",
```

Comment:
Could we populate scientific name also in Organism?

Reply:
Okay, I can do it.. Can you please explain me what difference it will make to test we are doing? sorry, it is not clear for me.